Sample_v3/Samples/CombineSimpleChat/CombineSimpleChannelsViewController.swift

```diff
@@ -236,8 +236,7 @@ class CombineSimpleChannelsViewController: UITableViewController {
         else { return }
         
         usersViewController.userListController = chatClient
-            // TODO: Change filter to all users after Filter adjustments
-            .userListController(query: .init(filter: .autocomplete("name", with: "a")))
+            .userListController(query: .init())
```

Comment:
Maybe `.init()` should be the default parameter of `. userListController()` then?

Reply:
I thought about it, but probably we want developers always use query explicitly, even if it default one.